.github/workflows/prerelease.yml

```diff
@@ -198,4 +183,16 @@ jobs:
           integration: nri-${{ env.INTEGRATION }}
           packageLocation: repo
           stagingRepo: true
-          upgrade: false
\ No newline at end of file
+          upgrade: false
+
+  notify-failure:
```

Comment:
thanks for finding this! I would update the release job as well, no?

Reply:
I included it in the release job here: f85201d0aa525f4d739e96508498ecce76f053d1, thanks for pointing it out!